halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/spinnaker/v1/service/Front50Service.java

```diff
@@ -82,9 +81,7 @@ public Type getType() {
     PersistentStore.PersistentStoreType type =
         deploymentConfiguration.getPersistentStorage().getPersistentStoreType();
     S3PersistentStore store = deploymentConfiguration.getPersistentStorage().getS3();
-    if (type == PersistentStore.PersistentStoreType.S3
-        && !StringUtils.isEmpty(store.getAccessKeyId())
-        && !StringUtils.isEmpty(store.getSecretAccessKey())) {
+    if (type == PersistentStore.PersistentStoreType.S3) {
```

Comment:
I find it dangerous to assume that the user is OK with propagating their hal credentials to the deployed services.  For example, if hal retrieves my personal IAM user creds, then it's not safe to assume I also want front50 running as 'me'.  How can I disable this to let front50 run with EC2 instance creds instead of my own creds?  It seems like there should be a configuration flag to enable this new behavior.